client/components/RelatedProducts/Related/RelatedProductsList.jsx

```diff
@@ -1,10 +1,17 @@
-import React from 'react';
+import React, {useState} from 'react';
```

Comment:
I see importing useState here but not using it, I assume it's for later usage ☺️